src/runtime_src/core/common/shim/buffer_handle.h

```diff
@@ -87,6 +89,18 @@ class buffer_handle
   bind_at(size_t /*pos*/, const buffer_handle* /*bh*/, size_t /*offset*/, size_t /*size*/)
   {
   }
+
+  virtual void
+  sync_aie_bo(xrt::bo&, const char *, xclBOSyncDirection, size_t, size_t)
+  {
+    throw xrt_core::error(std::errc::not_supported, __func__);
+  }
+
+  virtual void
+  sync_aie_bo_nb(xrt::bo&, const char *, xclBOSyncDirection, size_t, size_t)
+  {
```

Comment:
ditto

Reply:
bo instance is getting used in many "Aie" functions, we can cleanup with separate PR